Dockerfile

```diff
@@ -31,10 +31,11 @@ RUN apt-get install -y \
 # COPY PATHFINDER
 ARG VERSION 
 RUN mkdir /var/www/pathfinder
-RUN git clone --branch $VERSION https://github.com/exodus4d/pathfinder.git /var/www/pathfinder
+RUN git clone https://github.com/exodus4d/pathfinder.git /var/www/pathfinder
```

Comment:
Why did you take out the branch checkout?

Reply:
It was causing failed clones when I tried to build the image on my end

Reply:
You need to pass an argument in. `docker build <whatever> --build-arg VERSION=master`

Reply:
This definitely needs to stay so that we can provide images for previous versions (e.g 1.5.4)

Reply:
that --branch $Version results in an error when trying to build a new container (until the official on is updated)
it gives me /var/www/pathfinder is not a repository which is odd